.travis.yml

```diff
@@ -17,22 +17,23 @@ matrix:
     - brew list python >/dev/null 2>&1 || brew install python
```

Comment:
I added the encrypted  'BINTRAY_KEY' and 'BINTRAY_USER' with 
```
travis encrypt BINTRAY_KEY=[secret] --add env.global
```
which also did all the other reformatting.